utils/test-manager.js

```diff
@@ -48,7 +52,12 @@ class TestManager {
         gasLimit: 4700000,
         chainId: 3,
       };
-      return new etherlime.InfuraPrivateKeyDeployer(this.accounts[0].signer.privateKey, "ropsten", APIKEY, defaultConfigs);
+      return new etherlime.InfuraPrivateKeyDeployer(
+        this.accounts[0].signer.privateKey,
+        "ropsten",
+        APIKEY,
+        defaultConfigs,
+      );
     }
```

Comment:
I had problems running the tests on Kovan (all tests with relayed transactions failing). I tracked the problem to the provider not being properly configured.

It is fixed by creating an InfuraProvider when the network is Kovan:
```
    else if (this.network === "kovan") {
      const defaultConfigs = {
        gasPrice: 20000000000,
        gasLimit: 4700000,
        chainId: 42,
      };
      return new etherlime.InfuraPrivateKeyDeployer(
        this.accounts[0].signer.privateKey,
        "kovan",
        process.env.KOVAN_INFURA_KEY,
        defaultConfigs,
      );
    }
```

Reply:
Changed. I decided to have the kovan test scripts pass the correct Infura-based deployer to the constructor of `TestManager` to avoid creating it twice.